build.sh

```diff
@@ -139,6 +139,14 @@ if [[ "${CIRCLE_PROJECT_USERNAME}" == "PitchBlackRecoveryProject" ]]; then
     elif [[ $PB_OFFICIAL == 'true' ]]; then
         ./vendor/utils/pb_deploy.sh OFFICIAL $VENDOR $CODENAME
     fi
+else
+export BUILDFILE=$(find $(pwd)/out/target/product/${CODENAME}/PBRP*.zip 2>/dev/null)
```

Comment:
Add Odin tar file support too. Adapt fom this reference: https://github.com/PitchBlackRecoveryProject/vendor_utils/blob/e11d555b2c61a1a7cb48cd21fdaa33356266b0ff/pb_deploy.sh#L214-L219